app/js/lib/foxbox/network.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+// Private members.
+const p = {
+  // Private properties.
+  // Whether we can connect to the box via a local connection.
```

Comment:
__thinking aloud:__ :loud_sound: Honestly I'd put those comments in the constructor where actual variables are declared using these ```labels```, but up to you.